hudi-common/src/main/java/org/apache/hudi/common/table/timeline/CommitMetadataSerDe.java

```diff
@@ -29,6 +30,8 @@
  */
 public interface CommitMetadataSerDe extends Serializable {
 
+  TimelineLayoutVersion getLayoutVersion();
```

Comment:
Looks like this interface does not belong here.

Reply:
Removed.